### PR TITLE
updated page banners, upated home page, CSS cleanup

### PR DESCRIPTION
**Pull request by mmmavis**
Updated page banners, updated home page, CSS cleanup

Fixes https://github.com/MozillaFoundation/2016.Mozillafestival.org/issues/97
Fixes https://github.com/MozillaFoundation/2016.Mozillafestival.org/issues/93

@alanmoo review please? This seems like a big PR but that's because I had to apply same code/image changes for every single page.

### CODE REVIEW

File: components/hero-unit.jsx [hunk 1]
```diff
@@ -29,10 +29,24 @@ var HeroUnit = React.createClass({
   },
 
   render: function() {
+    var backgroundLines = [
```
Comment (alanmoo) [inline components/hero-unit.jsx]: Can you add a comment about what this is doing?

File: pages/home.jsx [hunk 2]
```diff
@@ -25,38 +25,37 @@ var Home = React.createClass({
...
-          <div>Join us to build, debate, and explore the future of our lives online.</div>
         </HeroUnit>
         <div className="centered content wide">
-          <h1>Buy a Ticket for MozFest 2016</h1>
-          <p>Join leaders of the open Web -- artists, activists, educators, scientists, journalists, policy-makers and technologists -- who are building a better internet.</p>
-          <Link to="tickets" className="button"><span>Buy a Ticket</span></Link>
+          <h1>More ways to participate</h1>
+          <p>The MozFest 2016 speaker series and keynotes will be broadcast live.</p>
+          <p>Sign up for email alerts and ways to participate remotely.</p>
+          <Link to="tickets" className="button"><span>Sign Up (FIXME)</span></Link>
```
Comment (alanmoo) [inline pages/home.jsx]: Fixme

File: components/hero-unit.jsx (hunk 1 above)
Comment (mmmavis) [inline components/hero-unit.jsx]: done!

File: pages/home.jsx (hunk 2 above)
Comment (mmmavis) [inline pages/home.jsx]: heh fixme fixed.

**Comment by mmmavis** [comment]
🎉 tada

**Pull request merged**